packages/orchestrator-ui-components/src/rtk/slices/orchestratorComponentOverride.ts

```diff
@@ -0,0 +1,23 @@
+import { Slice, createSlice } from '@reduxjs/toolkit';
+
+import { ValueOverrideFunction } from '@/components/WfoSubscription/overrides/useSubscriptionDetailValueOverride';
+
+export type ValueOverrideConfiguration = Record<string, ValueOverrideFunction>;
+
+export type OrchestratorComponentOverride = {
+    subscriptionDetail?: {
```

Comment:
This is fine for now but I wonder if we get more overrides if you can't make this a bit more generic. So perphaps make this something like 
```{
  [string]: valueOverrides 
}```

Keep something like that in mind when adding more overrides

Reply:
Will take this in consideration for the future overrides (next stories)